src/widgets/Autocomplete/Suggestion.ts

```diff
@@ -0,0 +1,38 @@
+import styled, { css } from 'styled-components'
+
+import { grey100, primary100 } from '@/theme/colors'
+
+import Flex, { Props as FlexProps } from '@/components/Flex'
+
+interface Props
+  extends FlexProps {
+  active?: boolean
```

Comment:
lets change all boolean props to start with `is` prefix
```suggestion
  isActive?: boolean
```